docs/GettingStarted.md

```diff
@@ -2479,7 +2479,7 @@ server {
 }
 ```
 
-Then you must enable the request queuing feature, by setting `request_queuing: true`, in the integration handling the request. For Rack-based applications, see the [documentation](#rack) for details.
+Then you must enable the request queuing feature, by setting `:request_queuing` to either `:include_request` or `:exclude_request`, in the integration handling the request. For Rack-based applications, see the [documentation](#rack) for details.
```

Comment:
My suggestion is that we clarify what options are available and what they mean. Perhaps in this section we could include a table that maps each value to a description of the behavior/span structure generated, just so its clear what "exclude request" means.